examples/00-mapdl-examples/composite_dcb.py

```diff
@@ -0,0 +1,400 @@
+# opposite side of the plate the displacement conditions are applied to
+# simulate the interfacial crack opening. These conditions are applied to the
+# top and bottom nodes corresponding to the geometrical edges located at
+# x = 0.0, z = 0.0 and x = 0.0, z = 3.4 respectively. Two different components
+# are assigned to these sets of nodes for a faster identification of the nodes
+# bearing reaction forces.
+
+# applying the two displacement conditions
+mapdl.allsel()
+mapdl.nsel(type_="s", item="loc", comp="x", vmin=0.0, vmax=0.0)
+mapdl.nsel(type_="r", item="loc", comp="z", vmin=3.4, vmax=3.4)
+mapdl.d(node="all", lab="uz", value=10)
+mapdl.cm("top_nod", "node")
+
+mapdl.allsel()
+mapdl.nsel(type_="s", item="loc", comp="x", vmin=0.0, vmax=0.0)
+mapdl.nsel(type_="r", item="loc", comp="z", vmin=0.0, vmax=0.0)
+mapdl.d(node="all", lab="uz", value=-10)
+mapdl.cm("bot_nod", "node")
+
+# applying the fix condition
+mapdl.allsel()
+mapdl.nsel(type_="s", item="loc", comp="x", vmin=85.0, vmax=85.0)
+mapdl.d(node="all", lab="ux", value=0.0)
+mapdl.d(node="all", lab="uy", value=0.0)
+mapdl.d(node="all", lab="uz", value=0.0)
+
+mapdl.eplot(
+    plot_bc=True, bc_glyph_size=3, title="", background="white", show_axes=False
+)
+
+###############################################################################
+# Solve the non-linear static analysis
+# ====================================
+#
+# A non-linear static analysis is run to in this very last step. One hundreds
+# substeps are requested in this case to have a smooth crack opening
+# progression as well as to facilitate convergency for the static solver.
+
+# entering the solution processor and defining the analysis settings
+mapdl.allsel()
+mapdl.finish()
+mapdl.run("/SOLU")
+mapdl.antype("static")
+
+# activating non-linear geometry
+mapdl.nlgeom("on")
+
+# requesting substeps
+mapdl.autots(key="on")
+mapdl.nsubst(nsbstp=100, nsbmx=100, nsbmn=100)
+mapdl.kbc(key=0)
+mapdl.outres("all", "all")
+
+# solving
+output = mapdl.solve()
+print(output)
+
+###############################################################################
+# Post-processing
+# ===============
+# The post-processing is perfored using PyMAPDL and PyDPF.
+#
+#
+# Post-processing of the results using PyMAPDL
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This section illustrates how to perform a quick prost-processing of the
+# the results using PyMAPDL. In this particular case as one of the key part is
+# measuring the delamination length, the cohesive damage parameter is plotted.
+# Although, the damage parameter is an element parameter, the result is
+# provided in terms of nodal result. Therefore, in this case we present the
+# result for just one of the four-noded cohesive element i.e. NMISC = 70.
+# The result for the other nodes are present at NMISC = 71,72,73.
+# The actual damage parameter nodal values can also be retrieved from the
+# solved model in form of table (or array).
```

Comment:
Interesting. 

So what we are plotting is the damage at one node of the whole element. It will be nice to plot this value for each node.... but because it is going to be difficult to identify each node with its corresponding ``NMISC``, the only reasonable thing to do might be to average the nodal values (taking the values given in the following ``pretab`` command) and plot those.

Reply:
Ok. We can leave this as it is at the moment then.